src/borg/helpers/parseformat.py

```diff
@@ -183,6 +183,7 @@ def replace_placeholders(text):
     data = {
         'pid': os.getpid(),
         'fqdn': socket.getfqdn(),
+        'reverse-fqdn': '.'.join(reversed(socket.getfqdn().split('.'))),
```

Comment:
as getfqdn can take rather long, please refactor this so it gets only called once and the result is used for both placeholders.

Reply:
I'll push a commit to call `getfqdn()` once per invocation of `replace_placeholders()` – if we were going to cache things more globally (i.e. across invocations of `replace_placeholders()`) we should probably do the same for `getpid()`, `gethostname()`, and `getuid()`, shouldn't we?